modules/ballerina-builtin/src/main/java/org/ballerinalang/nativeimpl/task/ScheduleAppointment.java

```diff
@@ -0,0 +1,97 @@
+/*
+*   Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+package org.ballerinalang.nativeimpl.task;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.ballerinalang.bre.Context;
+import org.ballerinalang.model.types.TypeKind;
+import org.ballerinalang.model.values.BFunctionPointer;
+import org.ballerinalang.model.values.BInteger;
+import org.ballerinalang.model.values.BString;
+import org.ballerinalang.model.values.BStruct;
+import org.ballerinalang.model.values.BValue;
+import org.ballerinalang.natives.AbstractNativeFunction;
+import org.ballerinalang.natives.annotations.Argument;
+import org.ballerinalang.natives.annotations.Attribute;
+import org.ballerinalang.natives.annotations.BallerinaAnnotation;
+import org.ballerinalang.natives.annotations.BallerinaFunction;
+import org.ballerinalang.natives.annotations.ReturnType;
+import org.ballerinalang.util.codegen.cpentries.FunctionRefCPEntry;
+
+/**
+ * Native function ballerina.model.task:scheduleAppointment.
+ */
+@BallerinaFunction(
+    packageName = "ballerina.task",
+    functionName = "scheduleAppointment",
+    args = {@Argument(name = "onTrigger", type = TypeKind.ANY),
+            @Argument(name = "onError", type = TypeKind.ANY),
+            @Argument(name = "appointmentScheduler", type = TypeKind.STRUCT, structType = "AppointmentScheduler")},
+    returnType = {@ReturnType(type = TypeKind.INT), @ReturnType(type = TypeKind.ANY)},
+    isPublic = true
+)
+@BallerinaAnnotation(annotationName = "Description", attributes = {@Attribute(name = "value",
```

Comment:
We don't need this set of annotations (BallerinaAnnotation). This was earlier used for API generation, but is no longer used.
Same for all the other classes.